indexclient/client.py

```diff
@@ -415,6 +415,12 @@ class DocumentDeletedError(Exception):
     pass
 
 
+class UrlMetadata(NamedTuple):
```

Comment:
I think typing.NamedTuple is only available in Python3.6+

Reply:
It's available for 3.5 as well but you do have to use [a special syntax](https://docs.python.org/3/library/typing.html#typing.NamedTuple) for creating the new class. It's rather moot tho cuz 2.7 only supports the [namedtuple](https://docs.python.org/2.7/library/collections.html#collections.namedtuple) from the collections module.